engines/ccgost/gost_ameth.c

```diff
@@ -532,6 +532,11 @@ static int pub_encode_gost01(X509_PUBKEY *pub, const EVP_PKEY *pk)
         ptype = V_ASN1_SEQUENCE;
     }
     order = BN_new();
+    if (!order)
+    {
```

Comment:
This is a small set of changes.  So a single new commit that goes against master is easiest to get review.  Close this PR, or rebase and push a new commit, whatever you prefer.  (Please use ==NULL styles of tests.)

Thanks for your help in making OpenSSL better!